pkg/controller/packet/server/device/managed.go

```diff
@@ -160,7 +169,14 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (resource.Ex
 
 	d.Status.ID = device.ID
 
-	// TODO: connection details
+	for _, n := range device.Network {
```

Comment:
Since we do this in the `Observe()` we don't need to do so here. This `TODO` comment can be removed though! Not sure why I put it here as I was actually referring to connection details to connect to the instance, which should be propagated to a Kubernetes `Secret` in `Observe()` (see example [here](https://github.com/crossplaneio/stack-gcp/blob/48dc96d9685094e9a1fc3f2eab321991fa46f33d/pkg/controller/cache/managed.go#L135)). Not sure exactly what we would want to put into the secret for a device, SSH info, ports, IP?

Reply:
I would go for now with public addresses IPv4 and IPv6, we can also surface the public ssh_key but there is a catch. When creating a device the device get injected with several SSH keys, if configured, so we could end up with several public SSH keys. 

I'd go with IPv4 only. What do you think?

Reply:
hmm if we are going to reflect IP addresses in the status, maybe we don't need to propagate a secret?

Reply:
In this case I don't see any use in propagating public SSH keys. I'd stick with just IPv4 and nothing else.

Reply:
I've left only IPv4.